src/ussdapp.js

```diff
@@ -40,75 +50,127 @@ go.app = function() {
 
     // START STATE
 
-        self.add('state_start', function(name) {
+        self.add("state_start", function(name) {
             // Reset user answers when restarting the app
             self.im.user.answers = {};
-            return self.states.create('state_check_registered');
+            return self.states.create("state_check_registered");
         });
 
         // interstitial to check registration status
-        self.add('state_check_registered', function(name) {
+        self.add("state_check_registered", function(name) {
             return go.utils
-                .get_or_create_identity({'msisdn': self.im.user.addr}, self.im, null)
+                .get_or_create_identity({"msisdn": self.im.user.addr}, self.im, null)
                 .then(function(identity) {
                     if(identity.details && !identity.details.registered) {
-                        self.im.user.set_answer('user_id', identity.id);
-                        return self.states.create('state_id');
+                        self.im.user.set_answer("user_id", identity.id);
+                        return self.states.create("state_facility_code");
                     } else {
-                        return self.states.create('state_already_registered');
+                        return self.states.create("state_check_quiz_status");
                     }
                 });
         });
 
         // EndState
-        self.add('state_already_registered', function(name) {
+        self.add("state_already_registered", function(name) {
             return new EndState(name, {
                 text: questions[name],
-                next: 'state_start'
+                next: "state_start"
             });
         });
 
     // REGISTRATION STATES
 
         // FreeText st-01
-        self.add('state_id', function(name) {
+        self.add("state_facility_code", function(name) {
             return new FreeText(name, {
                 question: questions[name],
                 next: function(input) {
```

Comment:
If we're not doing anything in the `next` functions, we should just simplify and set the next state as the value